GDJS/Runtime/events-tools/cameratools.ts

```diff
@@ -450,6 +450,16 @@ namespace gdjs {
         return runtimeScene.getLayer(layer).getDefaultZOrder();
       };
 
+      export const getLayerHighestZOrder = function (
```

Comment:
I understand why it was added here, but it's not related to camera.
Should we add a `layertools.ts` with only this function for now?

Reply:
You're right, I'll move it

Reply:
Sorry, actually, there is already a lot of function unrelated to camera so it can stay here.
The file should probably be renamed one day.